styles/components/_carousel.scss

```diff
@@ -54,7 +54,7 @@
 
     &-arrow {
       color: black;
-      font-family: 'Fira Code', monospace;
+      font-family: 'Work Sans', monospace;
```

Comment:
I know that this is how we are replacing the font-family right now, but isnt it time to just replace it in html tag once and remove these from all the other css files?

Reply:
I agree with that, we can try to define [Bulma sass variable](https://bulma.io/documentation/customize/variables/#generic-variables) or maybe customize [font family](https://bulma.io/documentation/helpers/typography-helpers/#font-family)

Reply:
I've tried to follow the [Bulma sass vairable](https://bulma.io/documentation/customize/variables/#generic-variables) way, lmk if something needs to be changed.